pymc3/distributions/dist_math.py

```diff
@@ -365,6 +365,7 @@ def conjugate_solve_triangular(outer, inner):
             grad = tt.triu(s + s.T) - tt.diag(tt.diagonal(s))
         return [tt.switch(ok, grad, floatX(np.nan))]
 
+
 class SplineWrapper (theano.Op):
```

Comment:
I've fixed all these issues. But just to argue about PEP8... :)

PEP8 forbids spaces after functions or methods, but doesn't say about a whitespace before the base class:
> Avoid extraneous whitespace in the following situations:
> ... (nothing interesting)
> Immediately before the open parenthesis that starts the argument list of a function call
> ... (nothing interesting)

Also it says that
> Method definitions inside a class are surrounded by a single blank line

so a blank line before the first method in a class is legitimate as it surrounds a class method, it is even possible to argue that it is required by PEP8 there.

Actually `pep8` linter seems to agree with both ways:

```sh
$ cat t.py 
class Foo (object):

    def foo(self):
        print('hi')


class Bar(object):
    def bar(self):
        print('hi')
$ pep8 t.py
$ # no errors reported
```

And as PEP8 is a formalized description of the code style used in Python standard library, we can check what is used in it, and it turns out that both variants are present there:

*  [class ZipInfo (object)](https://github.com/python/cpython/blob/master/Lib/zipfile.py#L294)
and [class LZMAFile(_compression.BaseStream)](https://github.com/python/cpython/blob/master/Lib/lzma.py#L38), although `class Foo(object)` is more frequent than `class Foo (object)`;

* [class _Outcome(object):\n    def __init__(self, result=None):](https://github.com/python/cpython/blob/master/Lib/unittest/case.py#L44) but [class _BaseTestCaseContext:\n\n    def __init__(self, test_case):](https://github.com/python/cpython/blob/master/Lib/unittest/case.py#L128), although ```class Foo:

    def __init__(...)
```
is more frequent than
```class Foo:
def __init__()
```

But it is just to argue, I've commited the fixes and I see that we need to not just follow PEP8, but also be consistent with PyMC3 :)

As a bit offtopic, I actually think that it would be nice to have something like `pep8` or `flake8` running automatically in Travis and failing builds in case of coding style violations. However I briefly investigated how much of the code should be changed to pass them and it seems like there are some violations and it might be necessary to agree on parts of pep8 that we would like to ignore or alter (like maximum line length), so I think that it should be discussed before.